### ISSUE TITLE
king-phisher service fails to start (the 2 first times) on a fresh install

### ISSUE THREAD

**Issue by sbrun** [securestate/king-phisher#417]
## Issue Description
I updated the king-phisher package for Kali. I tested it on a fresh installation. When I tried to start the king-phisher service, it failed a first time with the error "failed to create the postgresql user". 
If I tried to start again the service, it failed with the error "failed to create the postgresql database".
But user and database are created correctly and when I tried to start a third time the service, it run.

## Reproduction Steps
1. On a fresh installation, run systemctl start king-phisher
2. The service fails to start twice.

## Environment Details
Host OS: kali-rolling
King Phisher Version: 1.15.0 (and 1.14.0 IIRC)

## Error Details / Stack Trace
First time:
```
failed to create the postgresql user
```
Second time:
```
failed to create the postgresql database
```

## More information
I think the tests ```if rows != ['...'] ```  are not correct. Here is the patch I applied in Kali package:

```
--- a/king_phisher/server/database/manager.py
+++ b/king_phisher/server/database/manager.py
@@ -433,7 +433,7 @@ def init_database_postgresql(connection_url):
                if not is_sanitary(connection_url.password):
                        raise errors.KingPhisherInputValidationError('will not create the postgresql user (password contains bad characters)')
                rows = _popen_psql("CREATE USER {url.username} WITH PASSWORD '{url.password}'".format(url=connection_url))
-               if rows != ['CREATE ROLE']:
+               if rows[0] != 'CREATE ROLE':
                        logger.error('failed to create the postgresql user')
                        raise errors.KingPhisherDatabaseError('failed to create the postgresql user')
                logger.debug('the specified postgresql user was successfully created')
@@ -444,7 +444,7 @@ def init_database_postgresql(connection_url):
                if not is_sanitary(connection_url.database):
                        raise errors.KingPhisherInputValidationError('will not create the postgresql database (name contains bad characters)')
                rows = _popen_psql("CREATE DATABASE {url.database} OWNER {url.username}".format(url=connection_url))
-               if rows != ['CREATE DATABASE']:
+               if rows[0] != 'CREATE DATABASE':
                        logger.error('failed to create the postgresql database')
                        raise errors.KingPhisherDatabaseError('failed to create the postgresql database')
                logger.debug('the specified postgresql database was successfully created')
```

**Comment by zeroSteiner** [comment]
Oh there might be a new element in the rows that are returned. Thanks for the details, I'll check this out.

**Comment by zeroSteiner** [comment]
I am able to reproduce this, it looks like a bug.

**Comment by zeroSteiner** [comment]
Fixed in commit f95f9da294db96e109777100e0edeb997c953055. Thanks for reporting this issue.
